e2e-tests/portal/cypress/e2e/pages/utilisation-report-service/record-corrections/correctionSent.js

```diff
@@ -0,0 +1,7 @@
+const page = {
+  emailText: () => cy.get('[data-cy="email-text"]'),
+  emailListItem: (email) => cy.get(`[data-cy="email-list"] li:contains(${email})`),
```

Comment:
Pages file should just be selectors and not assertions. can this just be the data-cy.  the contains logic should happen in the test -  `correctionSent.emailList().contains(email)`

Reply:
This is technically a selector - it gets the list item which contains the email if there is one and then in the test the assertion checks the element exists. This just returns null if there is not a li with the content. That said I can change it if you still want me to?